pyquil/quilbase.py

```diff
@@ -236,10 +236,18 @@ def get_qubits(self, indices: bool = True) -> Set[QubitDesignator]:
     def qubits(self):
         return list(self.get_qubits(indices=False))
 
+    @qubits.setter
+    def qubits(self, qubits: Iterable[Union[Qubit, QubitPlaceholder, FormalArgument]]):
+        quil_rs.Gate.qubits.__set__(self, _convert_to_rs_qubits(qubits))
```

Comment:
This syntax may look weird, but it's necessary. If you are like me, you might think that something like this would work:

```py
super().qubits = ...
```

But it doesn't! Turns out that syntax is only good for getting attributes, not setting them. This workaround was shamelessly stolen from [this decades old](https://github.com/python/cpython/issues/59170) issue on the topic of setting superclass attributes via `super()`.